products/meilisearch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Meilisearch
+category: server-app
+tags: meilisearch
+iconSlug: meilisearch
+permalink: /meilisearch
+versionCommand: meilisearch --version
+releasePolicyLink: https://www.meilisearch.com/docs/learn/update_and_migration/versioning
+changelogTemplate: https://github.com/meilisearch/meilisearch/releases
+releaseDateColumn: true
+
+identifiers:
+-   repology: meilisearch
+
+auto:
+-   git: https://github.com/meilisearch/meilisearch.git
+
+releases:
+-   releaseCycle: "1.5"
+    releaseDate: 2023-11-20
+    eol: false
+    latest: "1.5.1"
+    latestReleaseDate: 2023-12-13
+
+---
+
+> [Meilisearch](https://www.meilisearch.com/) is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users.
```

Comment:
It's a bit too much commercial IMO. Could you rewrite this so this is more factual / neutral. Something like :

> [Meilisearch](https://www.meilisearch.com/) is a fast and scalable search engine that enables developers to integrate search capabilities into their websites and applications.

Reply:
Changed the description